engine/src/witness/btc/vault_swaps.rs

```diff
@@ -204,7 +203,7 @@ mod tests {
 	static MOCK_SWAP_PARAMS: LazyLock<UtxoEncodedData> = LazyLock::new(|| UtxoEncodedData {
 		output_asset: cf_primitives::Asset::Dot,
 		output_address: EncodedAddress::Dot(MOCK_DOT_ADDRESS),
-		parameters: SharedCfParameters {
+		parameters: BtcCfParameters {
```

Comment:
this may not be a straight forward task. The Btc version uses different Refund parameters:
btc: retry_duration + min output (u128
others: retry_duration + refund_address + price (u256)

While we can technically include the min_output as U256, and leave the refund_address as "default" (empty or invalid address that will never be read), the interfacing doesn't look good.

this is also the reason the `refund parameter` was part of the `vault swap extra parameters`. 
i think it would take some thinking to come up with an all-encompassing clean solution for this

Reply:
Bitcoin also uses slightly different encoding that the others, and for example doesn't include the broker_id.